bubble-wrap-style.yaml

```diff
@@ -2374,60 +2256,102 @@ layers:
                             family: Montserrat
                             weight: 500
                             fill: [0.308,0.308,0.308]
-                            size: [[7, 7px], [12, 8px], [15, 11px]]
+                            size: [[7, 7px], [13, 9px], [15, 10px]]
                             stroke: { color: global.text_stroke, width: 0px }
 
             default_priority_highway:
                 filter:
                     all:
+                        - kind: highway
                         - kind_detail: motorway
-                        - $zoom: { min: 9 }
                 draw:
                     mapzen_icon_library:
                         priority: 47
                         #color: yellow
                         visible: global.sdk_road_shields
-
+                early:
+                    filter:
+                        - $zoom: { max: 7 }
```

Comment:
Another minor thing, but in all these cases where we have a single filter condition, it's more succinct to not put it in an array, e.g.

```
filter:
  - $zoom: { max: 7 }
```

can just be

```
filter:
  $zoom: { max: 7 }
```

In the former, the engine is creating an unnecessary array filter for multiple conditions, even though there's only one (unlikely we'd ever see any real-world performance impact from this though).